data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2406 @@
+			choice
+				`	(Sure.)`
+				`	(Later.)`
+					defer
+			`	Curious to check on the huge ships you had seen under construction before, you go to the shipyard. There, you find an Unfettered warlord and go near him to learn more about them. "It looks like even the monkeys cannot resist the attraction of these ships," he says.`
+			choice
+				`	"Are these Shield Beetles? They do not really look like it."`
+				`	"I am simply curious. Why do you call me monkey, have I not earned your respect yet?"`
+					goto curious
+			`	"Not at all, they are something much more powerful, something so strong it would put to shame even your famous 'carriers'. We were producing Shield Beetles here for quite some time already, long before your pitiful existence."`
+			choice
+				`	"Why can I not buy Shield Beetles here then?"`
+				`	"Nothing is better than a carrier."`
+					goto brag
+			`	"Until now we did not allow you to purchase our biggest ships because we do not have the capacity to build enough of them at the same time. But you may help us with that, actually."`
+				goto mighty
+			
+			label brag
+			`	"Stupid monkey, thinks he knows everything... There are higher powers in this galaxy, and even then that is not required to match that carrier, because even if it is big and impressive, we have made something more advanced."`
+				goto mighty
+			
+			label curious
+			`	He was about to say something, but instead ignores your question.`
+			
+			label mighty
+			`	"If you want to help us and get access to our mighty shield beetles, go to <destination> where they make most if not all of their big ships and gather intel on how they mass produce them."`
+			choice
+				`	"Will the Hai will let me land on their home world?"`
+			`	"Why would you need a permission? Discretely land in an isolated region, whilst nobody is looking. If they detect you, just pretend you have a navigation issue."`	
+			choice
+				`	"I have a better idea: I will tell them I come to collect tribute."`
+				`	"This seems like it could easily go wrong, I am not interested."`
+					decline
+			`	He seems genuinely impressed, even more than he should be. These Unfettered are probably too used to being able to take and do what they want. "That is a brilliant idea! Well what are you waiting for? Go on then."`
+				accept
+	
+	on fail
+		dialog `You have failed an essential Unfettered mission. You may want to reload.`
+	on complete
+		log `Gathered intel on a Hai shipyard for the Unfettered, hopefully I will be able to buy Shield Beetles from them soon.`
+		payment 250000
+		conversation
+			`The warlord from earlier hands you <payment>.`
+			choice	
+				`	"Does this mean I can buy Shield Beetles now?"`
+				`	"What about my pay for the tribute?"`
+					goto tribute
+			`	He seems amused. "For that you would have to do a bit more than bringing some intel, we still do not have more capacity to build ships. But that will come soon, I first have to do some managing."`
```

Comment:
This would be a good spot to give the upgraded SB with 36400 shields under my idea above.

Reply:
I see what you mean but you haven't done enough to impress the warlord yet, this was just a test of your loyalty to his cause not your overall combat capacities